src/elements/kw-social-comment/kw-social-comment.html

```diff
@@ -0,0 +1,194 @@
+                font-size: 16px;
+                line-height: 43px;
+            }
+            
+            .comments-box .inputcomment {
+                @apply --layout-horizontal;
+                @apply(--layout-center);
+                width: 80%;
+                margin: 32px;
+            }
+            
+            .comment-avatar {
+                @apply --layout-flex-1;
+                margin-left: 16px;
+                margin-right: 16px;
+            }
+            
+            .comments-box .inputcomment .commentbox {
+                @apply --layout-horizontal;
+                @apply(--layout-center);
+                border-radius: 3px;
+                border: 1px solid var(--color-grey-lighter);
+                height: 45px;
+                width: 90%;
+                padding: 8px;
+                background-color: var(--color-grey-lightest);
+            }
+            
+            .comments-box .inputcomment .commentbox input {
+                width: 80%;
+                height: 32px;
+                font-size: 14px;
+                border: none;
+                margin: 8px;
+            }
+            
+            .comments-box .line {
+                height: 1px;
+                width: 90%;
+                display: block;
+                background-color: var(--color-white);
+                opacity: 0.2;
+            }
+            
+            .comment {
+                width: 80%;
+                height: 100px;
+                @apply --layout-horizontal;
+                @apply(--layout-center);
+            }
+            
+            .content {
+                font-size: 16px;
+                font-family: "Bariol";
+                color: var(--color-grey-dark);
+                width: 100%;
+            }
+            
+            .content p b {
+                margin-right: 6px;
+            }
+            
+            .content p {
+                margin-top: 6px;
+                margin-bottom: 6px;
+            }
+            
+            .content date-util {
+                font-size: 14px;
+                font-family: "Bariol";
+                color: var(--color-grey-dark);
+            }
+            
+            .comments-box p {
+                font-family: "Bariol";
+                color: var(--color-grey-dark);
+                font-size: 16px;
+            }
+            
+            iron-scroll-threshold {
+                @apply --layout-flex;
+                @apply --layout-vertical;
+            }
+        </style>
+        <div class="comments-box">
+            <h3>Be the first to comment!</h3>
+            <div class="line"></div>
+            <template is="dom-repeat" items="{{comments}}">
+                <div class="comment">
+                    <div class="comment-avatar"><img src="/assets/icons/shares/commentAvatar.svg" /></div>
+                    <div class="content">
+                        <p><b>{{item.author.username}}</b>{{item.text}}</p>
```

Comment:
I would use a class here instead of b tag (the styling use of the tag is deprecated) https://developer.mozilla.org/en/docs/Web/HTML/Element/b